pulpcore/cli/common/__init__.py

```diff
@@ -57,6 +57,8 @@ def _config_callback(ctx: click.Context, param: Any, value: Optional[str]) -> No
                             config[key] = new_config[key]
                 except FileNotFoundError:
                     pass
+                except PermissionError:
+                    pass
```

Comment:
```suggestion
                except (FileNotFoundError, PermissionError):
                    pass
```
I think, that's the proper syntax. Also we could consider using https://docs.python.org/3/library/contextlib.html#contextlib.suppress .